hiredis.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: hiredis
+  version: 1.2.0
+  epoch: 0
+  description: Minimalistic C client for Redis
+  copyright:
+    - license: BSD-3-Clause
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - openssl-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/redis/hiredis
+      tag: v${{package.version}}
+      expected-commit: 60e5075d4ac77424809f855ba3e398df7aacefe8
+
+  - runs: |
+      make USE_SSL=1 PREFIX=${{targets.destdir}}/usr install
+
+  - uses: strip
+
+subpackages:
+  - name: hiredis-dev
+    description: "hiredis dev headers"
+    pipeline:
+      - uses: split/dev
+
+update:
+  enabled: true
+  github:
+    identifier: redis/hiredis
```

Comment:
```suggestion
  github:
    identifier: redis/hiredis
    strip-prefix: v
    use-tag: true
```

Reply:
hey @ajayk where do I learn more about this update section. There are many things I am not sure what they do like `strip-prefix`, `tag-filter` etc. [Chainguard Academy](https://edu.chainguard.dev/open-source/wolfi/building-a-wolfi-package/#the-update-section) doesn't describe much.